### DISCUSSION TITLE
CS2 playback stuck on home menu

### DISCUSSION THREAD

**Discussion by ChengHeEric** [akiver/cs-demo-manager#687] [Q&A] [answered]
I can download and analyze the demos, but when I choose to watch demo, the game will start and entre a map loading page, after that I just go back to the home player lounge in cs2.

**Comment by akiver** [comment, marked as answer]
This is a CS2 issue, see the [doc](https://cs-demo-manager.com/docs/faq/playback#cs2-demo-playback-doesnt-start-or-crashes) for details.
